cdi/src/main/java/io/smallrye/concurrency/cdi/context/propagation/JtaContextProvider.java

```diff
@@ -0,0 +1,129 @@
+package io.smallrye.concurrency.cdi.context.propagation;
+
+import org.eclipse.microprofile.concurrent.ThreadContext;
+import org.eclipse.microprofile.concurrent.spi.ThreadContextProvider;
+import org.eclipse.microprofile.concurrent.spi.ThreadContextSnapshot;
+
+import javax.enterprise.inject.spi.CDI;
+import javax.transaction.InvalidTransactionException;
+import javax.transaction.SystemException;
+import javax.transaction.Transaction;
+import javax.transaction.TransactionManager;
+import java.util.Map;
+import java.util.logging.Logger;
+
+public class JtaContextProvider implements ThreadContextProvider {
```

Comment:
I am not sure why this is appearing as a new file since I just used drag and drop in my IDE to move it to a different maven module